dashboard/app/views/shared/_user_header.html.haml

```diff
@@ -42,16 +42,21 @@
 
 :javascript
   function hideUserOptions() {
```

Comment:
Agreed.  This is legacy code that I will leave for later examination when we start considering what to do with the user menu.  Ideally we can stop ajaxing that into the site, but it will require pegasus to have access to significantly more user information, with caching changes accordingly.  (Another possibility is to only ajax in the dropdown when opened, but we'd still need pegasus to be aware of username for the dropdown button "Hi X" text.)